app/src/main/java/com/example/next_show/models/Show.java

```diff
@@ -89,11 +88,13 @@ public Show(String title, String overview, String id, String network, int year_a
         List<Show> updated = new ArrayList<>();
 
         // turn them into my own Show objects for use later on
-        for(Show pre: parseShows){
-            Show currentShow = new Show(pre.getParseTitle(), pre.getParseOverview(), pre.getParseTVID(),
-                    pre.getParseNetwork(), pre.getParseYearAired(), pre.getParseGenres());
+        for(Show pShow: parseShows){
```

Comment:
```
pShow
```
isn't great variable naming -- what does the p mean?  Perhaps just calling it parseShow or show would be better.  We generally want to avoid non-common abbreviation usage in code